src/reset.css

```diff
@@ -0,0 +1,5 @@
+body,
+body * {
+  margin: 0;
```

Comment:
Don't reset margins using * selector. Better specify the exact tags you want to reset